jdbc/src/main/java/com/interface21/jdbc/core/JdbcTemplate.java

```diff
@@ -14,4 +20,63 @@ public class JdbcTemplate {
     public JdbcTemplate(final DataSource dataSource) {
         this.dataSource = dataSource;
     }
+
+    public Connection getConnection(){
+        try {
+            return dataSource.getConnection();
+        } catch (SQLException e) {
+            log.error(e.getMessage(), e);
+            throw new CannotGetJdbcConnectionException(e.getMessage(), e);
+        }
+    }
+
+    public void update(String sql, Object... params){
+        try(Connection conn = getConnection();
```

Comment:
```suggestion
    public void update(String sql, Object... params) {
        try (Connection conn = getConnection();
```
전반적으로 코드 포맷팅이 필요해 보여요!